docker/run_autostart.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+# Runs a docker container based on the exomy image that
+# autostarts every time ExoMy is started
+
+docker run \
+    -it \
+    -v $(pwd)/ExoMy_Software:/root/exomy_ws/src/exomy \
```

Comment:
```suggestion
    -v ~/ExoMy_Software:/root/exomy_ws/src/exomy \
```
using the pwd variable it is depending on the location from where we start the script.

Starting it from ~/ExoMy_Software/docker does't work.

My suggestion requires that the ExoMy_Software folder is in the ~ directory which we can enforce by changing the clone command in the instructions to include the correct directory.